### ISSUE TITLE
EarTrumpet manifest incorrectly modified without our knowledge

### ISSUE THREAD

**Issue by riverar** [microsoft/winget-pkgs#7836]
While preparing for an update to our EarTrumpet manifest, I discovered our manifest had been incorrectly altered (https://github.com/microsoft/winget-pkgs/pull/5759) and signed off by @KevinLaMS without any involvement on our part. This resulted in the distribution of ⚠ development binaries to all our winget users between December 29, 2020 and February 17, 2021.

Some questions:
* How do we identify affected users?
* How many times was EarTrumpet downloaded in this period?
* Can you clarify policy regarding manifest ownership, manifest updates, and describe how external contributions are vetted/approved?

cc: @aclinick

New PR: https://github.com/microsoft/winget-pkgs/pull/7533

**Comment by denelon** [comment]
@riverar does it look like #100 would address your concern here?

**Comment by riverar** [comment]
@denelon Yep, looks like it'll work for our needs. Are there any updates to the proposal? It's pretty old.

**Comment by denelon** [comment]
@riverar We're going to take a deeper dive over the next few weeks. I'll share progress/changes when we're about to start working on it. I'm sure there will be some questions, and we're happy to hear any feedback. I'll ping back here so we can get EarTrumpet locked down.

**Comment by pbrandstetter** [comment]
@riverar  I am sorry for the troubles you have now.
It was/is currently not clear who "owns" a manifest and since this is opensource ppl contribute and try to keep the system up to date.
I created #5759 because the manifest you provided did not work anymore. #5757 requested the problem and I tried to solve it (since nobody else did). In my opinion, if an "owner" of a manifest changes the update url she also has to change the manifests with the old urls.
Sorry again, hopefully we can do better next time.

**Comment by Samuel12321** [comment]
@pbrandstetter @denelon 
Hi, I'm one of the developers for ModernFlyouts, while preparing to release an updated manifest for 0.9.1, happened to notice commit c0a527438060d7ad96c3920588ec059ffd49b938 and discovered that **someone altered the link to the home page to a website that is not ours.** (Redirecting users to a 3rd part site not affiliated to ModernFlyouts)
I hope this issue is taken seriously and some solution can be found to prevent this.


@riverar sorry for hijacking this this issue but i thought they both stemmed from the same issue.

**Comment by riverar** [comment]
@Samuel12321 How dare you! Kidding. Microsoft's proposal is promising, do check that out.

@pbrandstetter Appreciate you reaching out, sounds like we're on the right path now.

**Comment by pbrandstetter** [comment]
@riverar Thank you for your answer. I will be more carefully next time. Hope msft will find a solution for manifests with owners so such problems won't occur again.

**Comment by denelon** [comment]
@pbrandstetter have you already updated the manifest to point to the proper site? The feature to specify owners for packages should help prevent others from making changes to manifests in the future.

**Comment by pbrandstetter** [comment]
@denelon riverar already submitted #7533 with the most current version. This pr also points to the correct address.

**Comment by riverar** [comment]
The proposal is almost a year old now, are there any updates? Is it going to be executed on? Timeline?

**Comment by denelon** [comment]
It's next up for one of our engineers. After we review the plan and adjust, they will get assigned and we'll update the specification. It's going to be the first of several features that we would like to link to publicly visible meta-data. Part of the solution will involve how we can properly identify package owners. I'd like to get the initial implementation done this month if possible.

**Comment by denelon** [comment]
@riverar is your GitHub alias the only one you want approved to submit PRs for https://github.com/microsoft/winget-pkgs/tree/master/manifests/f/File-New-Project/EarTrumpet/2.1.8.0?

Would you like this restricted to the package `manifests/f/File-New-Project/EarTrumpet/*`

-or-

Would you like this to apply for any package published by `manifests/f/File-New-Project`

**Comment by denelon** [comment]
We still have the "business process" work to build out for identifying the proper owners for a package, and therefore the manifests. As that work completes, we will share the results with the community.

**Comment by riverar** [comment]
@denelon Would prefer to take over the whole org (option 2) but both options look good.

**Comment by denelon** [comment]
@riverar we've completed the technical implementation. We're working on the business process portion of the verified developer feature. Can you send me an e-mail so I can get permission to use EarTrumpet as a first implementation?

**Comment by riverar** [comment]
@denelon Will do.

**Comment by vedantmgoyal2009** [comment]
Any updates on this?

**Comment by o-l-a-v** [comment]
> Any updates on this?

https://github.com/microsoft/winget-pkgs/issues/100

**Comment by WouterTinus** [comment]
I will +1 this given the fact that someone recentely published a package for my project without my knowledge or concent, even though it wouldn't have been difficult to contact me and I'd have been willing to put in the effort myself (in fact that's how I found out). 

My software is highly security sensitve (it runs as `SYSTEM` and manages TLS certificates on servers) so the **last** thing that users need for something like that is a frickle supply chain. I'm sure that you do some AV-scan and a human moderator gives the manifest a glance, but that nothing a sophisticated attacker wouldn't be able to bypass. For now I see no security breach, but it's highly concerning that anyone can just submit and modify packages via PR.

https://github.com/microsoft/winget-pkgs/issues/265975

**Comment by WouterTinus** [comment]
> https://github.com/microsoft/winget-pkgs/blob/master/Tools/Auth.csv

That looks promising, can anyone PR to that? Or how does one get on that list?

**Comment by riverar** [comment]
@denelon Hey Demitrius, if you need anything from me just poke me. 👋
